bluesky/magics.py

```diff
@@ -19,162 +19,50 @@
     from toolz import partition
 
 
-# Wrap the user namespace to provide newbie-friendly KeyError messages.
-class Namespace:
-    def __init__(self, ns):
-        self.ns = ns
-
-    def __getitem__(self, key):
-        try:
-            return self.ns[key]
-        except KeyError:
-            raise KeyError("No variable named {!r} is defined in the "
-                           "user namespace. Define it and try again."
-                           "".format(key))
-
 @magics_class
 class BlueskyMagics(Magics):
 
+    RE = RunEngine({})
```

Comment:
Probably better to pass it a new loop?  Do not want this to conflict with the main RE at all.